src/Starlight/System.hs

```diff
@@ -89,12 +90,12 @@ neighbourhoodOf c sys@System{ bodies, characters } = sys
     B (Star _) -> True
     _          -> received > threshold
     where
-    r = distance (a^.actor_.position_) (c^.actor_.position_)
-    received = Watts ((c^.ship_.radar_.power_.unitary * gain * aperture * (a^.magnitude_) * patternPropagationFactor ** 4) / ((4 * pi) ** 2 * prj (r ** 4)))
+    r = qd (a^.actor_.position_) (c^.actor_.position_)
+    received = Watts ((c^.ship_.radar_.power_.unitary * gain * aperture * prj (a^.actor_.magnitude_) * patternPropagationFactor ** 4) / ((4 * pi) ** 2 * prj (r ** 2)))
   patternPropagationFactor = 1
-  gain = 1000
+  gain = 1000000
```

Comment:
There was a unit conversion error (km -> m) in the previous definition of `magnitude_` for bodies which meant that we needed another three zeroes here.